pom.xml

```diff
@@ -340,23 +383,23 @@
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-artifact</artifactId>
-            <version>3.0.5</version>
+            <version>3.5.2</version>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
-            <version>3.0.5</version>
+            <version>3.5.2</version>
```

Comment:
we have 3.5.2 dependency here but above we only declare 3.1.1 as maven version?

Reply:
adjusted to 3.1.1 for now. Not sure how connected prerequiste and maven version must be linked. But for now it is not  mandatory to jump for last version